tests/test_client.py

```diff
@@ -26,3 +32,144 @@ def test_list_with_params(index_client):
             break
     if not found:
         raise AssertionError()
+
+
+def test_get_latest_revision(index_client):
+    """
+    :type index_client: indexclient.client.IndexClient
+    """
+    hashes = {'md5': 'ab167e49d25b488939b1ede42752458c'}
+    doc = index_client.create(
+        hashes=hashes,
+        size=10,
+        file_name="removable.txt",
+        urls=["s3://service.hidden.us/lostinspace"]
+    )
+    latest = index_client.get_latest_revision(doc.did)
+    if latest.did == doc.did or latest.file_name == doc.file_name or latest.hashes == doc.hashes:
+        pass
+    else:
+        raise AssertionError()
+
+
+def test_invalid_input(index_client):
+    """
+    :type index_client: indexclient.client.IndexClient
+    """
+
+    with pytest.raises(HTTPError):
```

Comment:
Great candidate for parametrization. Pass tuples (ExepectedException, latest_revision)